simulator/test_simulator.py

```diff
@@ -0,0 +1,270 @@
+import unittest
+
+from cart_pole.interface import Config, Error, State
+from cart_pole.simulator import CartPoleSimulator, PhysicalParams
+
+
+def eval_path_length(v, a, n, dt):
+    '''
+    Evaluate path with uniform acceleration and discrete speed control.
+    Args:
+        * v - start velocity
+        * a - acceleration
+        * n - step number
+        * dt - time step
+    '''
+    t = n * dt
+    return v * t + (t + dt) * t / 2 * a
```

Comment:
where `(t + dt)` comes from? i thought it should be `a * t * t / 2`

Reply:
Velocity changes discreetly  (stepped profile).